src/realm/object-store/shared_realm.cpp

```diff
@@ -902,6 +902,8 @@ void Realm::close()
     m_transaction = nullptr;
     m_binding_context = nullptr;
     m_coordinator = nullptr;
+    m_scheduler = nullptr;
+    m_config.scheduler = nullptr;
```

Comment:
Should this instead set `m_config` an empty value? Seems a bit awkward that we're relying on the Realm instance to clean up fields in its config.

Reply:
Good question! `m_config` is a `struct`, so emptying it is not as trivial as setting it to a `nullptr`.  As it is established on construction, I am not sure what would be the right way to tear it down would be, except for doing something like `m_config = {null, "", 0, false, true, ... etc}`.  Maybe it would make sense to set all `shared_ptr` values within the `struct` to `nullptr`?

Reply:
I thought you could just do `m_config = {}`.

Reply:
Clearing it should be just `m_config = {};` to reset everything to the default values.

Reply:
That worked!  Thanks for the suggestions.  I'm still brushing up on c++.